### DISCUSSION TITLE
Wiki-like Knowledge Base

### DISCUSSION THREAD

**Discussion by CodeByChaim** [Budibase/budibase#8615] [Help] [answered]
Can I build a wiki-like Knowledge Base via  budibase

**Comment by ttelik** [comment, marked as answer]
In principle, sure. In practice, I believe you can find other frameworks that are well set up for hosting wikis out-of-the-box.

For any engineering project, a good place to start is to write down a list of features, then think high-level about how to implement them. One example that comes to mind is parsing user-entered text for formatting and links. I'm not certain how to do this in BB, and you need to be very careful about security when treating user-entered code as "executable".

**Reply by CodeByChaim** [reply]
basically I am looking for an open source knowledgebase (like wiki or confluence) that I can integrate into my existing React app via REST API's
